### PR TITLE
Unsafe: refactor UNSAFE_ methods from /common/ 

### PR DESCRIPTION
**Pull request by gitstart-augur**
#4341  
- Fixed initial reverted PR(Made some static class methods instance methods in error).

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🥇
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/5m3h33q**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Comment by bthaile** [comment]
image

File: packages/augur-ui/src/modules/common/form.tsx [hunk 1]
```diff
@@ -1765,14 +1781,17 @@ export class Input extends Component<InputProps, InputState> {
...
+    }
+    else {
+      return null
     }
   }
 
-  UNSAFE_componentWillUpdate(nextProps, nextState) {
+  componentDidUpdate(prevProps, prevState) {
     if (
       nextProps.canToggleVisibility &&
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.tsx]: nextProps is no longer defined. Just bring up the Ui would have found this in about 1 second

**Review by bthaile — CHANGES_REQUESTED** [review]
need to test before submitting PR

**Comment by pgebheim** [comment]
@gitstart-augur -- Did that force push address @bthaile's issue?

**Comment by pgebheim** [comment]
Is this being fixed in #4695, #4964, #4696 ?

**Comment by gitstart-augur** [comment]
Yes @pgebheim  this has been fixed.

**Comment by pgebheim** [comment]
@bthaile mind doing a review?

**Comment by bthaile** [comment]
@bconfortin can you review this PR. Make sure to pull it down and click around

**Review by bconfortin — CHANGES_REQUESTED** [review]
TextInput is breaking and Input is not working as intended.
After you fix those two, I suggest taking 5-10 minutes testing and checking if you don't have any console errors.

The other components seems to be working properly. Good job.

**Comment by bthaile** [comment]
I'm thinking too much context is needed to work these refactor tasks.

**Comment by gitstart-augur** [comment]
@bthaile  @bconfortin  We fixed breaking TextInput. Also took some time to go through the whole site while having my console open(Should do that going forward). 
For Upcoming PR's i will be asking questions in the channel for test context i miss in case of any.

**Comment by bconfortin** [comment]
TextInput and Input seems to be working now. Nice job.

**Review by bconfortin — APPROVED** [review]

**Review by bthaile — APPROVED** [review]
Lgtm

**Pull request merged**